src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -152,4 +189,12 @@ public static boolean matches(MaxwellFilter filter, String database, String tabl
 			return filter.matches(database, table);
 		}
 	}
+
+	public static boolean matchesValues(MaxwellFilter filter, String database, String table, Map<String, Object> data) {
+		if (filter == null) {
+			return true;
+		} else {
+			return filter.matchesValues(data);
+		}
+	}
```

Comment:
nope, I think you just accidentally disabled all the rest of maxwell filtering.  Please have `MaxwellFilter#matches` call `MaxwellFilter#matchesValues` instead.  (you'll need to change the signature of `matches`)

Reply:
Hi,
Thanks for the comment.
Actually the MaxwellFilter.matchesValues() is invoked from AbstractReplicator.shouldOutputRowMap(), which is finally called from BinlogConnectorReplicator.getTransactionRows().

In method BinlogConnectorReplicator.getTransactionRows(), the shouldOutputEvent() is called first in order to filter the database and table, then in the for loop of the rows, shouldOutputRowMap() is called in order to filter the row base on the values.

If we do the row filter in MaxwellFilter.matches(), we have to change the signature of the method to have row passed to this method, and it will be invoked once for each row, and it should have less performance than the current solution.

Reply:
ah, yeah, I missed that.  ok.